client/app/models/applicant.js

```diff
@@ -0,0 +1,5 @@
+import Model from '@ember-data/model';
```

Comment:
Rebase off develop to bring in the new applicant model

Reply:
Thanks!